index.js

```diff
@@ -149,7 +149,9 @@ var _isISO31661Alpha2 = _interopRequireDefault(require("./lib/isISO31661Alpha3")
 
 var _isBase = _interopRequireDefault(require("./lib/isBase32"));
 
-var _isBase2 = _interopRequireDefault(require("./lib/isBase64"));
+var _isBase2 = _interopRequireDefault(require("./lib/isBase58"));
```

Comment:
You can remove the unrelated changes on this file or just unstage it altogether. The rest looks good for landing.